content/applications/services/helpdesk/overview/ratings.rst

```diff
@@ -88,14 +92,14 @@ Manually hide individual ratings
 --------------------------------
 
 Individual ratings can be manually hidden from the portal. This allows for specific ratings to be
-kept out of the performance metrics that are shown to customers.
+kept out of the performance metrics shared with customers.
 
 To make a rating visible only to internal users, navigate to the page for a rating. This can be done
 in one of the following ways:
 
-   - Go to :menuselection:`Helpdesk --> Reporting --> Customer Ratings` and click on one of the
-     kanban cards for an individual rating.
-   - Navigate to :menuselection:`Helpdesk --> Tickets --> All Tickets` and remove the
+   - Go to :menuselection:`Helpdesk app --> Reporting --> Customer Ratings` and click on one of the
+     Kanban cards for an individual rating.
+   - Navigate to :menuselection:`Helpdesk app--> Tickets --> All Tickets` and remove the
```

Comment:
Maybe add another see also to the portal documentation

https://www.odoo.com/documentation/17.0/applications/general/users/portal.html